addons/l10n_lv/data/account_account_tag_data.xml

```diff
@@ -0,0 +1,1424 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+
+  <record id="account_account_tag_a" model="account.account.tag">
+    <field name="name">A. Aktīvs</field>
+    <field name="applicability">accounts</field>
```

Comment:
we should use account codes instead of tags for the reports

Reply:
We could do it on our side if you agree to not have to define all these tags.  Always good if it can be talked through with a product owner.

Reply:
These tags are necessary for Balance sheet and reports so it will be better to leave them as is.